ansible/roles/rhel_zero_touch_user/tasks/copy-contents.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Copy user's contents
+  when: rhel_zero_touch_user_copy_user_contents is defined
+  block:
+    - name: Copy user's content to resource dir
+      when: __content_r.resource is defined
+      ansible.builtin.copy:
+        content: "{{ __content_r.content }}"
+        dest: >
+          /home/{{ rhel_zero_touch_user_name }}/
+          {{ rhel_zero_touch_user_resource_dir_name }}/
+          {{ __content_r.resource }}
```

Comment:
Checking yaml to json conversion... this looks like it injects spaces and a trailing newline:
```
  "dest": "/home/{{ rhel_zero_touch_user_name }}/ {{ rhel_zero_touch_user_resource_dir_name }}/ {{ __content_r.resource }}\n"
```